src/app/pages/shared/bucket/bucket-lifecycle-datatable-page/bucket-lifecycle-datatable-page.component.ts

```diff
@@ -312,6 +312,7 @@ export class BucketLifecycleDatatablePageComponent implements OnInit {
             name: 'days',
             label: TEXT('Expires after'),
             value: _.get(rule, 'Expiration.Days', 365),
+            hint: TEXT('Expires after the specified days from object creation.'),
```

Comment:
from -> since ?

Reply:
Hmmm, after thinking about the sentence itself does not make much sense. What about `Expires after the specified days since the object was created.`?

Reply:
`Expiration days after creation` ?